### ISSUE TITLE
Add support for SKU Selector on Product Summary

### ISSUE THREAD

**Issue by juliomoreira** [vtex-apps/store-discussion#17]
Former Issue: [#129](https://github.com/vtex-apps/product-summary/issues/129)

**Describe the solution you'd like**
We would like to have the ability to render at least one SKU Selector (color is the most used one from our experience).
Typically this kind of control has side-effects on the summary itself updating the featured image price, buy button and other attributes.

**Describe alternatives you've considered**
Right now the only alternative we have is to extend the `product-summary` interface

**Additional context**
![Screenshot from 2019-06-27 14-34-20](https://user-images.githubusercontent.com/1207017/60287626-b8b1b780-98e8-11e9-94e4-21aad72858d0.png)

**Comment by klzns** [comment]
Feature implemented

**Comment by alextalha** [comment]
How can i be using color instead thumb image on the sku-selector? I didn't find this parameter in the documentation

**Comment by jaimesanchezgalvisRicoh** [comment]
Thank you for the implementation, but I have two questions.

Is it possible to display all available options in the storefront without the need to show two identical products?

How can the color feature be used in large inventories where it's not feasible to upload a color image for each product? Thank you.

![image](https://github.com/vtex-apps/store-discussion/assets/135970671/6aefe087-c1be-4efb-a0b0-f148b4de3e88)

**Comment by ganobrega** [comment]
@jaimesanchezgalvisRicoh 

Have you already made the following configuration in SKU registration?

https://developers.vtex.com/docs/guides/vtex-similar-products-variants

**Comment by jaimesanchezgalvisRicoh** [comment]
Thank you for the response, and I apologize for not commenting earlier that the issue had already been resolved. I appreciate your assistance. I have reorganized the catalog, and after the cache update, the issues were eventually resolved. Many thanks.

I have used this tutorial for a color selector, in case it can be helpful to someone else. https://developers.vtex.com/docs/guides/vtex-io-documentation-configuring-custom-images-for-the-sku-selector
